src/Evolution/Executables/GeneralizedHarmonic/DeadlockActions.hpp

```diff
@@ -0,0 +1,118 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <iomanip>
+#include <sstream>
+#include <string>
+
+#include "ControlSystem/Tags/SystemTags.hpp"
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "Evolution/DiscontinuousGalerkin/InboxTags.hpp"
+#include "Evolution/DiscontinuousGalerkin/MortarTags.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "Parallel/OutputInbox.hpp"
+#include "Parallel/Printf.hpp"
+#include "Time/Actions/ChangeSlabSize.hpp"
+#include "Time/Tags/Time.hpp"
+#include "Utilities/PrettyType.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace gh::deadlock {
+struct PrintControlSystemCurrentMeasurement {
```

Comment:
We don't generally put code other than the executables themselves in "Executables" directories.  I could be convinced, but I'd like a rationale (as a GitHub comment).

Reply:
I didn't really know where to put these. Right now only the BBH will need them so I considered just putting them in the metavars file, but that makes the file even more crowded and it's already so complicated. However, eventually we'll want to use these (or similar/modified versions) for other executables that can deadlock. The control system one can easily go in the control sys library, however, the element one pulls on a lot of things (and with subcell we'll probably want something related to that as well) so I didn't know where to put it. A new library in the `Evolution/` directory? But how to keep dependencies straight if we have subcell stuff? Suggestions are welcome

Reply:
That's reasonable.  We can figure out if this is the correct choice when we want to use it more.